pkg/issuer/acme/dns/execute/execute.go

```diff
@@ -0,0 +1,103 @@
+package execute
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"github.com/golang/glog"
+	"github.com/jetstack/cert-manager/pkg/issuer/acme/dns/util"
+	"os"
+	"os/exec"
+	"strings"
+	"time"
+)
+
+var workingDirectory = "/srv/execute_plugin_dir/"
```

Comment:
This directory should be configurable via CLI flags if possible - if you add it as a new option on the ACMEOptions struct in `pkg/controller`, you should be able to access it in the constructor function in dns.go to pass to the NewDNSProvider function.